build/verify-features.ts

```diff
@@ -53,12 +54,31 @@ function findError(filename: string): string | void {
 
 	const featureMeta = featuresInReadme.find(feature => feature.id === featureId);
 	if (!featureMeta) {
-		return `ERR: The feature ${featureId} should be described in the readme`;
+		return `ERR: ${featureId} should be described in the readme`;
 	}
 
 	if (featureMeta.description.length < 20) {
 		return `ERR: ${featureId} should be described better in the readme (at least 20 characters)`;
 	}
+
+	// eslint-disable-next-line @typescript-eslint/prefer-regexp-exec
+	const featureMatch = readmeContent.match(findFeatureRegex(featureId as FeatureID));
+	// eslint-disable-next-line @typescript-eslint/prefer-regexp-exec
+	const highlightedFeatureMatch = readmeContent.match(findHighlightedFeatureRegex(featureId as FeatureID));
+	if (
+		(featureMatch && featureMatch.length > 1) // If the description occurs more than once in the large list of features
+		|| (highlightedFeatureMatch && featureMatch) // If the description appears in both the feature list and the highlighted features section
+	) {
+		// Uncomment the following to print out line numbers
```

Comment:
I think unused code should be removed. If we change our minds, we can always get it back from this PR's commit.

Reply:
I just commented it out for now as I was waiting on the decision to include line numbers. If we decide to keep it, I will uncomment that part, else I will delete it completely.

Reply:
I've kept the line number functionality, see my comment below.